tests/compiletest.rs

```diff
@@ -113,39 +110,48 @@ fn test_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
 }
 
 fn run_tests(mode: Mode, path: &str, target: &str, with_dependencies: bool) -> Result<()> {
-    let mut config = test_config(target, path, mode, with_dependencies);
+    let config = test_config(target, path, mode, with_dependencies);
 
     // Handle command-line arguments.
     let mut after_dashdash = false;
-    config.path_filter.extend(std::env::args().skip(1).filter(|arg| {
-        if after_dashdash {
-            // Just propagate everything.
-            return true;
-        }
-        match &**arg {
-            "--quiet" => {
-                config.quiet = true;
-                false
-            }
-            "--" => {
-                after_dashdash = true;
-                false
+    let mut quiet = false;
```

Comment:
I can't find any place that this variable s actually used?

Reply:
oh no rustc, why hast thou forsaken me